client/debug/main.go

```diff
@@ -7,9 +7,9 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
```

Comment:
```suggestion
```

Should be grouped with cosmos imports. Can't multi line suggest otherwise id move it 😄